docs/spelling_wordlist.txt

```diff
@@ -0,0 +1,1366 @@
+classable
+classname
+classpath
+classpaths
+cli
+clientId
+cloudml
+cloudp
+cloudsqldatabehook
+cloudwatch
+cmd
+cmdline
+cmds
+cname
+cnf
+codebase
+codecov
+colour
+colours
+commiter
+comnand
+compaitible
+compat
+componensts
+concat
+conda
+conf
+config
+configMap
+configMapRef
+configMapsRefs
+configfile
+configmap
+configs
+configuing
+conftest
+conn
+connectTimeoutMS
+containerConfiguration
+contextmgr
+contrib
+coverals
+cp
+cpu
+cpus
+crd
+createDisposition
+creationTimestamp
+credssp
+criccomini
+cron
+cronjob
+crontab
+crypto
+csrf
+css
+csv
+ctor
+cubeName
+customDataImportUids
+dabf
+dagBag
+dagbag
+dagbags
+dagmodel
+dagre
+dagrun
+dagruns
+dask
+dat
+databricks
+datacenter
+datadog
+dataflow
+dataframe
+datapipe
+datapoint
+dataproc
+dataset
+datasetId
+datasets
+datasource
+datastore
+datasync
+datatypes
+datestamp
+datetime
+datetimes
+dbapi
+dbexportcmd
+dbimportcmd
+dbs
+dbtapquerycmd
+dbutils
+de
+decomissioning
+decrypt
+decrypted
+decyption
+deepcopy
+defence
+deidentify
+del
+delim
+dep
+deploymentUrl
+deps
+desc
+desecending
+deserialization
+deserialize
+deserialized
+deserializing
+dest
+detele
+dev
+devel
+df
+diclaimer
+dicts
+dimberman
+dimensionX
+dingding
+dir
+dirs
+discoverable
+displayName
+displyed
+distcp
+distro
+distros
+dns
+dnsPolicy
+dnspolicy
+docstring
+docstrings
+doesn
+doesnt
+donot
+dropdown
+druidHook
+ds
+dsn
+dttm
+dtypes
+dupplicate
+durations
+dynamodb
+eRevalue
+ec
+ecb
+ecoding
+editorconfig
+eg
+elasticsearch
+emplate
+emr
+emty
+enableAutoScale
+encodeed
+encryptor
+encyption
+enqueue
+entrypoint
+enum
+env
+envFrom
+envvar
+eols
+errno
+eslint
+etl
+europe
+eval
+evals
+evo
+exasol
+excecutor
+execvp
+exeuction
+exitcode
+explict
+exportingmultiple
+extensibility
+externalbrowser
+exts
+faas
+facebook
+failover
+faq
+fbee
+fc
+feedCard
+feluelle
+feng
+fernet
+fetchmany
+fetchone
+fileds
+filehandle
+fileloc
+filepath
+filesize
+filesystem
+filesystems
+filetype
+findall
+fluentd
+fmt
+fn
+fo
+fokko
+followsa
+formating
+formatter
+fot
+fqdn
+friedly
+frontend
+fs
+fsGroup
+func
+functionarity
+gRPC
+ga
+gantt
+gcloud
+gcp
+gcpcloudsql
+gcs
+gdbm
+generateUploadUrl
+getfqdn
+getframe
+getsource
+gid
+gif
+gitginore
+github
+gitignore
+gitpython
+gitter
+globstring
+glyphicon
+golang
+googlapiclient
+googleapiclient
+gpu
+gpus
+groupId
+hadoop
+hadoopcmd
+hashicorp
+hasn
+hcatalog
+hdd
+hdfs
+hdfscli
+hdp
+healthjump
+heartrate
+hiteshs
+hivecmd
+hiveconf
+hiveconfs
+hivemeta
+hiveserver
+honoured
+hostPath
+hostname
+hostnames
+hotfix
+houqp
+hql
+html
+htmlcontent
+http
+httpbin
+https
+hvac
+hyperparameter
+iPython
+iTerm
+iam
+idempotence
+idempotency
+ie
+ignorable
+ignoreUnknownValues
+imageORfile
+imagePullPolicy
+imageVersion
+imap
+img
+imgcat
+impyla
+incompliancies
+incorret
+infile
+infoType
+infoTypes
+inforamtion
+init
+initcontainer
+initcontainers
+initdb
+initialisation
+initialising
+initlialized
+instafail
+installable
+instanceTemplates
+instantiation
+instatiating
```

Comment:
used in `- [AIRFLOW-397] Documentation: Fix typo "instatiating" to "instantiating"`

Reply:
I went ahead and changed this